src/olympia/legacy_api/tests/test_utils.py

```diff
@@ -8,8 +8,8 @@ def test_compatible_version(self):
         addon = addon_factory()
         version = version_factory(
             addon=addon,
-            status=amo.STATUS_PUBLIC, version='99')
+            file_kw={'status': amo.STATUS_PUBLIC}, version='99')
         version_factory(
-            addon=addon, status=amo.STATUS_PUBLIC, version='100',
+            addon=addon, file_kw={'status': amo.STATUS_PUBLIC}, version='100',
```

Comment:
nit: `amo.STATUS_PUBLIC` is the default for `file_factory` so not needed in these two.

Reply:
Nice find, thanks! Fixed in eab651223c